server/src/controllers/eventController.service.ts

```diff
@@ -363,48 +361,80 @@ interface checkoutForm {
   seatingAcc: string;
   comments: string;
   optIn: boolean;
+  donorbadge?: boolean;
+  vip?: boolean;
+  volunteerlist?: boolean;
 }
 
 export const updateContact = async (
-    formData: checkoutForm,
     prisma: ExtendedPrismaClient,
+    contact: ValidatedContact,
+    check = 0,
```

Comment:
Do you think you could rename the `check` parameter to be a bit more descriptive of its use. And/or instead of using integers, we could use strings that explain what is being checked